src/fava/application.py

```diff
@@ -270,11 +270,19 @@ def fava_api_exception(error: FavaAPIException):
 
 @app.route("/")
 @app.route("/<bfile>/")
-def index():
+def index(bfile=None):
     """Redirect to the Income Statement (of the given or first file)."""
     if not g.beancount_file_slug:
         g.beancount_file_slug = next(iter(app.config["LEDGERS"]))
-    return redirect(url_for("report", report_name="income_statement"))
+    default_path = ""
+    if not bfile:
```

Comment:
I don't quite see why this is necessary - we need to prepend the slug in any case, no?

Reply:
I thought based on the `@app.route` lines before the definition of this function:
```
@app.route("/")
@app.route("/<bfile>/")
```
that the intended behavior was to handle redirection here even when the slug was already present. I found that if I unconditionally prepended it, I ended up with the slug in the URL twice if I visited a URL with the slug already present.

Maybe this is an artifact of how I'm doing redirection here?

Reply:
> I found that if I unconditionally prepended it, I ended up with the slug in the URL twice if I visited a URL with the slug already present.

That's odd, in my quick testing, `bfile` was always None (since it gets removed by the `_pull_beancount_file`) and always prepending it works.

Reply:
You're right, I'll remove this unnecessary logic. I'm not 100% sure what I was seeing before, but the only way I've been able to re-create what I was seeing before is to use a relative path. I suspect I falsely attributed a change.